### DISCUSSION TITLE
Desktop slow rendering

### DISCUSSION THREAD

**Discussion by daught1** [devemux86/cruiser#614] [Q&A] [answered]
Hello. I love the projector for finally being able to have a laptop at the basecamp to plan routes, but map rendering is very slow. Zooming and moving the map is very choppy. Any ideas if there is something I can do to improve? I am using arch with version 0.5-4.

**Comment by devemux86** [comment, marked as answer]
Hi @daught1,

Do you mean desktop Cruiser 5?

Do you use offline maps?
Which offline maps and with which map theme?

Can you post a video, so we can understand what you see?

Offline maps may take some ms to render their tiles.
